### ISSUE TITLE
lmh mine no longer works

### ISSUE THREAD

**Issue by kohlhase** [KWARC/localmh#213]
which is really a pity, since I need it now to install all mine on the new laptop.

**Comment by tkw1536** [comment]
1) Dev might be broken, try and use master
2) Error message? Details? I can not reproduce the bug. 
3) Workaround: Use ```lmh ls``` to list all installed repositories on the old machine and then do a lmh install on the new machine.

**Comment by kohlhase** [comment]
master did not work either, but gave an error ```cannot write on /``` It seems that it is not writing the file, probably because of path problems (at least, I can never find a file).

**Comment by tkw1536** [comment]
Yes, that broke when using docker. Please use lmh ls > /path/to/some/file

which should do the same thing.

**Comment by kohlhase** [comment]
I have done that, and that worked, now now I am trying to get lmh to run, ... 
and have some problems. But they can wait until you are here.
